javascript/phone-number.js

```diff
@@ -0,0 +1,36 @@
+var PhoneNumber = function(number) {
+  this.string = number;
+}
+
+PhoneNumber.prototype.number = function() {
+  var raw = this.string;
+  var final = "";
+
+  for (var i = 0; i < raw.length; i++) {
+    if (parseInt(raw[i]) >= 0) {
+      final += raw[i]
+    }
+  }
```

Comment:
It may be a simpler, more concise approach to utilize regex to remove non-digit characters.